.github/workflows/CI.yml

```diff
@@ -5,7 +5,7 @@ jobs:
     strategy:
       matrix:
         os: [macos-latest, ubuntu-latest, windows-latest]
-        node-version: [14]
```

Comment:
As we're on this we could drop the Cache Node.js modules task, too, and add `cache: 'npm'` to the setup-node action ([ref](https://github.com/bpmn-io/bpmn-js/blob/develop/.github/workflows/CI.yml#L19)).